perf/perf-shaping.cc

```diff
@@ -0,0 +1,66 @@
+#include <benchmark/benchmark.h>
+
+#include "hb.h"
+
+static void shape (benchmark::State &state,
+		   const char *text_path, unsigned text_length,
+		   hb_direction_t direction, hb_script_t script,
+		   const char *font_path)
+{
+  hb_blob_t *blob = hb_blob_create_from_file (font_path);
+  assert (hb_blob_get_length (blob));
+  hb_face_t *face = hb_face_create (blob, 0);
+  hb_font_t *font = hb_font_create (face);
+  hb_face_destroy (face);
+
+  hb_blob_t *text_blob = hb_blob_create_from_file (text_path);
+  assert (hb_blob_get_length (text_blob));
+  unsigned text_blob_length;
+  const char *text = hb_blob_get_data (blob, &text_blob_length);
+  assert (text_length < text_blob_length);
```

Comment:
I am not shaping the whole text here, maybe I should but that means not iteration and shaping only once.